### PR TITLE
Added examples for finder document scanner

### PR DESCRIPTION
**Pull request by matthewrindel**

### CODE REVIEW

File: ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs [hunk 1]
```diff
@@ -178,6 +178,19 @@ private void OnButtonClick(object sender, EventArgs e)
                 var intent = DocumentScannerActivity.NewIntent(this, configuration);
                 StartActivityForResult(intent, Constants.CAMERA_DEFAULT_UI_REQUEST_CODE);
             }
+            else if (button.Data.Code == ListItemCode.ScanDocumentWithFinder)
+            {
+                var configuration = new FinderDocumentScannerConfiguration();
+                configuration.SetCameraPreviewMode(CameraPreviewMode.FitIn);
+                configuration.SetIgnoreBadAspectRatio(true);
+                configuration.SetTextHintOK("Don't move.\nCapturing document...");
+                //configuration.SetFinderLineColor(Color.Blue);
```
Comment (scan-mayank) [inline ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs:187]: we can remove these comments

File: ScanbotSDKExamples/NET/ReadyToUseUI.iOS/Controller/MainViewController.cs
Comment (scan-mayank) [inline ScanbotSDKExamples/NET/ReadyToUseUI.iOS/Controller/MainViewController.cs:1]: Minor most point: 
There are a few unnecessary new lines, we can remove it.

**Comment by scan-mayank** [comment]
I got a few build-time errors, as the newer beta with FinderDocumentScanner is missing.

File: ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs (hunk 1 above)
Comment (matthewrindel) [reply ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs:187]: These are left as an example, but I have moved the configuration lines under the comment.

**Review by scan-mayank — COMMENTED** [review]
The code looks good. 
But there are a few new lines, comments which we can remove.

**Comment by matthewrindel** [comment]
> The code looks good. But there are a few new lines, comments which we can remove.

Thanks @scan-mayank, I've updated the packages to the latest beta and you can give it a final look. 

In terms of comments, I have updated the examples for what I have changed, but also the comments are often meant to be illustrative and to enable a developer to uncomment to see a different kind of change. So I've done my best to make it clearer that they are demonstrative, but this won't apply to any existing comments (which we can always look at for future changes).

**Pull request merged**